VERSION

```diff
@@ -1 +1 @@
-2.11.0
+2.12.0
```

Comment:
I *think* this is correct, though the instructions [here](https://github.com/grafana/mimir/blob/main/RELEASE.md#merging-release-branch-into-main) say to leave this as the current version in `main`. 🤔 I changed this because [the merge PR for 2.11 also changed it](https://github.com/grafana/mimir/pull/7002/files#diff-7b60b8e351cbb80c47459ffe2c79f1a26404871f49294780fe47ad0e58c09350R1).